src/routes/articles.js

```diff
@@ -4,14 +4,16 @@ const { logger } = require('../lib')
 const isAuthenticated = require('../middleware/isAuthenticated')
 const {
 	ArticleDoesNotExistForCommentError,
-	ArticleDoesNotExistError
+	ArticleDoesNotExistError,
+	TagAlreadyAssignedToPostError
 } = require("../services/errors")
 
 const log = logger()
 const router = express.Router()
 const ERROR_MAP = {
 	[ArticleDoesNotExistForCommentError.name] : 422 ,
-	[ArticleDoesNotExistError.name] : 404
+	[ArticleDoesNotExistError.name] : 404,
+	[TagAlreadyAssignedToPostError] : 400
```

Comment:
@beblicarl, 
```suggestion
	[TagAlreadyAssignedToPostError.name] : 400
```

also let's make it `422`. The client request is correct (not necessarily malformed) but the server cannot process it due to the constraint.

Reply:
oh okay, lemme make the change then

Reply:
done @jocrah